src/main/java/core/basesyntax/AreaCalculator.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface AreaCalculator {
```

Comment:
The interface name `AreaCalculator` should not be a verb. Consider renaming the interface to something that represents its purpose without implying action, like `AreaComputable` or `AreaProvider`.